### DISCUSSION TITLE
v0.27.0

### DISCUSSION THREAD

**Discussion by eginhard** [idiap/coqui-ai-TTS#455] [Versions & Releases]
### Features
* Speaker caching for cloned voices by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/438
  - For usage details see https://coqui-tts.readthedocs.io/en/latest/cloning.html
  - ⚠️ The old caching mechanism of Bark and Tortoise has been removed, switch to the new one instead.
* Provide `synthesize()` method with a common interface for every TTS model by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/453
  - ⚠️ Deprecate `speaker_id` argument of `synthesize()`, use `speaker` instead.
  - ⚠️ Deprecate `config` argument of `synthesize()`, it can safely be left out.
* Add OpenAI-compatible endpoint to the server by @teddybear082 in https://github.com/idiap/coqui-ai-TTS/pull/421
  See https://coqui-tts.readthedocs.io/en/latest/server.html#openai-compatible-endpoint

### Fixes
* Update coqui-tts-trainer to 0.3.0 to fix numerous training-related bugs by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/423
  For the full list of fixes see https://github.com/idiap/coqui-ai-Trainer/releases/tag/v0.3.0
* fix(configs): add default padding character by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/425
* Fix KeyError when speaker_id is empty string in TTS server by @mehulanshumali in https://github.com/idiap/coqui-ai-TTS/pull/436
* Fix: Update xtts finetuning Colab to support Gradio 5 by @eulphean in https://github.com/idiap/coqui-ai-TTS/pull/408
* Bring back compatibility with numpy1 by @MarwanMashra in https://github.com/idiap/coqui-ai-TTS/pull/413
* fix: update XTTS/Tortoise GPT code for HF transformers 4.52+ by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/414
* build: lower minimum pytorch version back to 2.1 by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/432
* refactor(phonemizer): replace mecab-python3 with fugashi for japanese by @eginhard in https://github.com/idiap/coqui-ai-TTS/pull/417
* docs: add Docker Compose config by @KishoOoOoOo in https://github.com/idiap/coqui-ai-TTS/pull/411

## New Contributors
* @KishoOoOoOo made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/411
* @eulphean made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/408
* @MarwanMashra made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/413
* @teddybear082 made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/421
* @mehulanshumali made their first contribution in https://github.com/idiap/coqui-ai-TTS/pull/436

**Full Changelog**: https://github.com/idiap/coqui-ai-TTS/compare/v0.26.2...v0.27.0

<hr /><em>This discussion was created from the release <a href='https://github.com/idiap/coqui-ai-TTS/releases/tag/v0.27.0'>v0.27.0</a>.</em>
